_pages/dev/css-architecture.md

```diff
@@ -2,74 +2,89 @@
 title: CSS Architecture
 ---
 
-This document provides a high level overview of the CSS architecture of Spartacus.
+This page provides a high level overview of the CSS architecture in Spartacus.
 
 ## Style Library
 
-Spartacus is provided as a set of standard components, distributed in so-called _npm libraries_. Libraries are used to ensure both extensibility and upgradability at the same time.
+Spartacus is provided as a set of standard components that are distributed in npm libraries. These libraries help to ensure that Spartacus remains both extensible and upgradable at the same time.
 
-To deliver full flexiblity for styling, all CSS rules are provided in a separate library (`@spartacus/styles`). This make the styles completely optional, configurable and extendible. Customers can extend or replace the standard styles as well as implement their own style rules.
+To deliver as much flexibility as possible for styling, all of the CSS rules are provided in a separate `@spartacus/styles` library. This makes the styles completely optional, configurable, and extensible. You can extend or replace the standard styles, and also implement your own style rules.
 
-Additionally, an existing UI framework such as Bootstrap can be used in Spartacus without making it a hardcoded dependency.
+Additionally, an existing UI framework, such as Bootstrap, can be used in Spartacus without making it a hard-coded dependency.
 
-### Style Library Versioning
+### Versioning
 
-Spartacus libraries support Semantic Versioning, which means that no breaking changes are allowed during a major version. Uncontrolled style changes in Spartacus would have a bad effect on customers, as unexpected styling could appear without notice.
+Spartacus libraries support semantic versioning, which means that breaking changes are not allowed during the life cycle of a major version. This is also true for the style library. A new style rule, or an adjusted rule, will influence the existing storefront experience. Customers who follow the semantic version scheme should not be surprised by the appearance of new styles during the life cycle of a stable release.
 
```

Comment:
In the paragraph above, in the first sentence it says that “breaking changes are not allowed during the life cycle of a major version”. In the final sentence of this same paragraph, it says “Customers who trust the semantic version scheme should not be surprised by the appearance of new styles during the life cycle of a stable release”… In this case, is there a difference between "major release" and "stable release"? Also, just so I understand the intent of this paragraph, is "the appearance of new styles" not considered a breaking change? I'm just trying to figure out if there is a contradiction between the first and last sentences here 🙂

Reply:
A major version is consider to be non-breaking. I indeed also refer to it as a stable release, but that's not really nice. We better stick to major version. The appearance of new styles is indeed a breaking change.

I could rephrase it a bit:

> Spartacus libraries support semantic versioning, which means that breaking changes are not allowed during the life cycle of a major version. This is also true for the style library; a change in de styles, that would lead to a difference in the UI experience, is considered a breaking change.
> 
> We expect you to rely on the semantic versioning scheme. Therefor, we avoid to add implicit changes to the style library during the life cycle of a major release.

Reply:
Perfect -- thank you!